commands/command_get.go

```diff
@@ -0,0 +1,75 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/github/git-lfs/git"
+	"github.com/github/git-lfs/lfs"
+	"github.com/spf13/cobra"
+)
+
+var (
+	getCmd = &cobra.Command{
+		Use:   "get",
+		Short: "get",
+		Run:   getCommand,
+	}
+)
+
+func getCommand(cmd *cobra.Command, args []string) {
+	var ref string
+	var err error
+
+	if len(args) == 1 {
+		ref = args[0]
+	} else {
+		ref, err = git.CurrentRef()
+		if err != nil {
+			Panic(err, "Could not get")
+		}
+	}
+
+	pointers, err := lfs.ScanRefs(ref, "")
+	if err != nil {
+		Panic(err, "Could not scan for Git LFS files")
+	}
+
+	q := lfs.NewDownloadQueue(lfs.Config.ConcurrentTransfers(), len(pointers))
+
+	for _, p := range pointers {
+		q.Add(lfs.NewDownloadable(p))
+	}
+
+	q.Process()
+
+	target, err := git.ResolveRef(ref)
+	if err != nil {
+	}
+
+	current, err := git.CurrentRef()
+	if err != nil {
+	}
+
+	if target == current {
```

Comment:
Does it make sense to return early if they aren't equal, and remove a level of nesting?